np.sh

```diff
@@ -15,7 +15,7 @@ if test "0" != "$(git rev-list --count --left-only @'{u}'...HEAD)"; then
 	exit 128;
 fi
 
-trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
+trashCli=$(node -p "require.resolve('.bin/trash')");
```

Comment:
If you install `np` globally, `require.resolve('.bin/trash')` will still look for `.bin/trash` in your local `node_modules` folder, not the global one. Test in any folder that doesn't have `trash-cli` installed locally.